src/cards/cards.css

```diff
@@ -1,7 +1,20 @@
 /* https://docs.google.com/presentation/d/1ORi0ZjFeOoNyK7l3R8ms9v0uSmLerUIUfG1fXh_XcTk/edit#slide=id.g5b283a7c15_0_86 */
+@media only screen and (min-width: 481px) {
```

Comment:
I don't think that cards should have a width in the styleguide. Instead, the login page can set the width on its card.

Reason why is that we need cards to be as reusable as possible.